mmdet3d/evaluation/functional/scannet_utils/evaluate_semantic_instance.py

```diff
@@ -25,8 +25,7 @@ def evaluate_matches(matches, class_labels, options):
     dist_confs = [options['distance_confs'][0]]
 
     # results: class x overlap
-    ap = np.zeros((len(dist_threshes), len(class_labels), len(overlaps)),
-                  np.float)
```

Comment:
Suggest to change `np.float` to `np.float32`

Reply:
Looks like the default dtype for np.zeros is `float` which according to numpy documentation means exactly the same as old `np.float`. Why do we want to specify `np.float32`?

Reply:
We expect to keep `dtype` to make the code more readable.